chartfx-math/src/main/java/de/gsi/math/DataSetMath.java

```diff
@@ -179,8 +178,8 @@ public static DataSet averageDataSetsIIR(final DataSet prevAverage, final DataSe
                     ((DoubleErrorDataSet) prevAverage2).set(i, newX, newY2, eyn, eyp);
                 }
             }
-            final double newEYN = Math.sqrt(Math.abs(newY2 - Math.pow(newY, 2)) + eyn * eyn);
-            final double newEYP = Math.sqrt(Math.abs(newY2 - Math.pow(newY, 2)) + eyp * eyp);
+            final double newEYN = java.lang.Math.sqrt(java.lang.Math.abs(newY2 - java.lang.Math.pow(newY, 2)) + eyn * eyn);
+            final double newEYP = java.lang.Math.sqrt(java.lang.Math.abs(newY2 - java.lang.Math.pow(newY, 2)) + eyp * eyp);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.abs' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)